<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainForm</class>
 <widget class="QMainWindow" name="MainForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mite</string>
  </property>
  <property name="windowIcon">
   <iconset resource="qml.qrc">
    <normaloff>:/earth.png</normaloff>:/earth.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>Standard</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QWidget" name="widgetTop" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <property name="spacing">
               <number>8</number>
              </property>
              <property name="leftMargin">
               <number>8</number>
              </property>
              <property name="topMargin">
               <number>8</number>
              </property>
              <property name="rightMargin">
               <number>8</number>
              </property>
              <property name="bottomMargin">
               <number>4</number>
              </property>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widgetBottom" native="true">
             <layout class="QVBoxLayout" name="verticalLayout_8">
              <property name="spacing">
               <number>8</number>
              </property>
              <property name="leftMargin">
               <number>8</number>
              </property>
              <property name="topMargin">
               <number>4</number>
              </property>
              <property name="rightMargin">
               <number>8</number>
              </property>
              <property name="bottomMargin">
               <number>8</number>
              </property>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>High</string>
          </attribute>
          <widget class="QScrollArea" name="scrollAreaHa">
           <property name="geometry">
            <rect>
             <x>290</x>
             <y>10</y>
             <width>231</width>
             <height>241</height>
            </rect>
           </property>
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOn</enum>
           </property>
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents_2">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>212</width>
              <height>239</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QWidget" name="widgetIoHa" native="true">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
          <widget class="QScrollArea" name="scrollAreaSa">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>10</y>
             <width>231</width>
             <height>241</height>
            </rect>
           </property>
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOn</enum>
           </property>
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>212</width>
              <height>239</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QWidget" name="widgetIoSa" native="true">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </widget>
         <widget class="QWidget" name="tab_3">
          <attribute name="title">
           <string>State</string>
          </attribute>
          <widget class="QScrollArea" name="scrollAreaState">
           <property name="geometry">
            <rect>
             <x>290</x>
             <y>0</y>
             <width>211</width>
             <height>251</height>
            </rect>
           </property>
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOn</enum>
           </property>
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents_3">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>192</width>
              <height>249</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QWidget" name="widgetIoState" native="true">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
          <widget class="QGroupBox" name="gbState">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>60</y>
             <width>121</width>
             <height>81</height>
            </rect>
           </property>
           <property name="title">
            <string>Locked</string>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
          <widget class="QGroupBox" name="groupBox">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>81</width>
             <height>41</height>
            </rect>
           </property>
           <property name="title">
            <string>Satellite</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QSpinBox" name="sbSatelliteIndex">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>32</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnSend">
              <property name="text">
               <string>Set</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QPushButton" name="btnPlot">
           <property name="geometry">
            <rect>
             <x>110</x>
             <y>10</y>
             <width>75</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Plot</string>
           </property>
          </widget>
          <widget class="QGroupBox" name="gbDevice">
           <property name="geometry">
            <rect>
             <x>150</x>
             <y>60</y>
             <width>117</width>
             <height>81</height>
            </rect>
           </property>
           <property name="title">
            <string>Device</string>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QComboBox" name="comboBox"/>
            </item>
            <item>
             <widget class="QPushButton" name="btnOpen">
              <property name="text">
               <string>Open</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QPlainTextEdit" name="teLog">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>280</y>
             <width>491</width>
             <height>91</height>
            </rect>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>128</height>
            </size>
           </property>
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOn</enum>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="qml.qrc"/>
 </resources>
 <connections/>
</ui>
